vit-servicing-station-lib/src/db/models/challenges.rs

```diff
@@ -30,9 +32,11 @@ impl Queryable<challenges::SqlType, Db> for Challenge {
         String,
         // 3 -> rewards_total
         i64,
-        // 4 -> fund_id
+        // 4 -> rewards_total
+        i64,
+        // 5 -> fund_id
         i32,
-        // 5 -> fund_url
+        // 6 -> fund_url
```

Comment:
Why not preserve the old order of these?

Reply:
Well, I tried to keep the rewards attributes close together. Just that.

Reply:
I just realize, the confusion may come because the 4# item was wrong it should have been `proposers_rewards`. I fixed it now.